tmcprototype/SdpSubarrayLeafNode/SdpSubarrayLeafNode.xmi

```diff
@@ -31,7 +31,6 @@
     <deviceProperties name="SdpSubarrayNodeFQDN" description="FQDN of the SDP Subarray Node Tango Device Server.">
```

Comment:
Property name should be SdpSubarrayFQDN.

Reply:
Done.

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd